package.json

```diff
@@ -6,13 +6,13 @@
     "url": "https://virtocommerce.com"
   },
   "license": "SEE LICENSE IN LICENSE.txt",
-  "version": "1.63.0",
+  "version": "2.0.0",
   "private": true,
   "type": "module",
   "scripts": {
     "dev": "vite",
     "dev:expose": "yarn dev --host",
-    "build": "yarn validate && yarn generate:routes && yarn build-only",
+    "build": "yarn validate && yarn build-only",
```

Comment:
do we still need script `generate:routes` below?

Reply:
no. I keep it to not forget to remove the whole service latter